.gitlab-ci.yml

```diff
@@ -368,95 +368,35 @@ init_workspace:
       - build_revisions.env
 
 .template_build_test_acc: &build_and_test_acceptance
-  image: teracy/ubuntu:18.04-dind-18.09.9
+  image: mendersoftware/mender-test-containers:qa-acceptance-testing
   needs:
     - init_workspace
   services:
   - docker:18-dind
   tags:
     - mender-qa-slave-highcpu
+  variables:
+    - RUN_INTEGRATION_TESTS=false
```

Comment:
I just took from the script https://github.com/mendersoftware/mender-qa/pull/444/files#diff-037ea159eb0a7cb0ac23b851e66bee30fb838ee8d0d99fa331a1ba65283d37f7L390 - thinking that might be an easy way to override it... if it's no longer used I'll happily remove it!

Reply:
Right. I think it's not, so you can try removing it and see if it causes problems.

Reply:
That export in the script is a leftover from when `yoctobuild-build.sh` script was able to run integration tests too. It can be safely removed in this rework, and we don't need it here neither.

Reply:
should now be gone 👌